lib/msal-core/.npmrc

```diff
@@ -0,0 +1 @@
+@azuread:registry=https://npm.pkg.github.com/AzureAD
```

Comment:
This tells npm to get `@azuread` scoped packages from the Github registry, instead of the main npm registry.